testing/camunda-process-test-java/src/main/java/io/camunda/process/test/impl/assertions/DecisionInstanceAssertj.java

```diff
@@ -40,12 +35,20 @@ public class DecisionInstanceAssertj
     implements DecisionInstanceAssert {
 
   private final CamundaDataSource dataSource;
+  private final DecisionMatchedRulesAssertj decisionMatchedRulesAssertj;
+  private final DecisionOutputAssertj decisionOutputAssertj;
```

Comment:
:wrench: The new assertion classes are only used by this decision assertion. There is no need for separate classes.